test/components/HabitTabs.spec.js

```diff
@@ -16,11 +16,11 @@ describe('(Component) TimeTabs', () => {
       habits: []
     }
 
-    const wrapper = shallow(<TimeTabs {...props} />);
+    const wrapper = shallow(<HabitTabs {...props} />);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Expected an identifier and instead saw '<'.<br>Expected ')' and instead saw 'HabitTabs'.<br>Missing semicolon.<br>'spread/rest operator' is only available in ES6 (use esnext option).<br>Expected an assignment or function call and instead saw an expression.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (32% scanned).